### PR TITLE
ttk - 1896 and 1897 stories

### PR DESCRIPTION
**Pull request by modusemerson**
admin tests cases
settlement admin test cases
quotes negative scenarios test cases
funds in test cases
funds out test cases

stories
https://github.com/mojaloop/project/issues/1896
https://github.com/mojaloop/project/issues/1897

Report
image

Issues

1. Api-tests/quotes there is an error an invalid request is returning 202 instead of 400 response code
image

2. Api-tests / Settlementwindow There are failures but they are not bugs, is just that we need a closed window and a settled window in our environment and currently we dont have any
image

### CODE REVIEW

**Review by vijayg10 — COMMENTED** [review]
Not able to get 100% pass for Quotes

File: collections/hub/golden_path/api-tests/SettlementWindows/settlement-admin.json [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+          "apiVersion": {
+            "minorVersion": 0,
+            "majorVersion": 1,
+            "type": "settlements"
+          },
+          "operationPath": "/settlementWindows/{id}",
+          "path": "/settlementWindows/50",
+          "method": "get",
+          "params": {
+            "id": "50"
```
Comment (vijayg10) [inline collections/hub/golden_path/api-tests/SettlementWindows/settlement-admin.json:21]: It should not be a hardcoded value

File: collections/hub/golden_path/api-tests/SettlementWindows/settlement-admin.json [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                "description": "Response status to be 200",
+                "exec": [
+                  "expect(response.status).to.equal(200)"
+                ]
+              },
+              {
+                "id": 2,
+                "description": "Check state for window is SETTLED",
+                "exec": [
+                  "expect(response.body[0].state).to.equal(\"SETTLED\")"
```
Comment (vijayg10) [inline collections/hub/golden_path/api-tests/SettlementWindows/settlement-admin.json:141]: I think we should check all the elements in the array. Ref postman

**Review by vijayg10 — COMMENTED** [review]
Funds out tests are not at 100% pass

**Comment by modusemerson** [comment]
> Not able to get 100% pass for Quotes
> Screenshot 2021-01-11 at 5 35 01 PM

Hi @vijayg10  could you pls show me what the error is?... I can see there is only one test failing for each test case... maybe is the same for all of them... could you pls take a screenshot of the failed tests?

**Comment by modusemerson** [comment]
Hi @vijayg10  I have push my changes to my local fork... I can see this PR was updated automatically, could you please review again?.. thanks in advance

**Comment by modusemerson** [comment]
closing this pr since we have another one created

**Pull request closed without merging**